rebasehelper/tests/conftest.py

```diff
@@ -59,3 +62,20 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mock_spec_object_generator(monkeypatch):
+    def _mock_spec(override_methods: Dict[str, Callable], set_attributes: Dict[str, Any]):
```

Comment:
What's the difference between method and attribute? Why not just use a single argument?